test/common/formatter/substitution_formatter_test.cc

```diff
@@ -534,7 +534,7 @@ TEST(SubstitutionFormatterTest, streamInfoFormatter) {
   {
     StreamInfoFormatter duration_format("DURATION");
     absl::optional<std::chrono::nanoseconds> dur = std::chrono::nanoseconds(15000000);
-    EXPECT_CALL(stream_info, requestComplete()).WillRepeatedly(Return(dur));
+    EXPECT_CALL(stream_info, currentDuration()).WillRepeatedly(Return(dur));
```

Comment:
Would it be possible to make a test that verifies that this is the in-flight duration? This is just testing that we're calling a specific function on StreamInfo

Reply:
I guess it would require some king of mocking to verify this, otherwise it's hard to get the exact time. I did add a test case though, please let me know what you think.